File: app/src/main/java/com/aar/app/wordsearch/data/network/responses/WordsUpdateResponse.java
package com.aar.app.wordsearch.data.network.responses;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WordsUpdateResponse {

    @SerializedName("update")
    private boolean mUpdate;
    @SerializedName("revision")
    private int mRevision;
    @SerializedName("data")
    private List<ThemeResponse> mData;

    public boolean isUpdate() {
        return mUpdate;
    }

    public void setUpdate(boolean update) {
        mUpdate = update;
    }

    public int getRevision() {
        return mRevision;
    }

    public void setRevision(int revision) {
        mRevision = revision;
    }

    public List<ThemeResponse> getData() {
        return mData;
    }

    public void setData(List<ThemeResponse> data) {
        mData = data;
    }
}

File: app/src/main/java/com/aar/app/wsp/data/network/responses/WordsUpdateResponse.kt
package com.aar.app.wsp.data.network.responses

import com.google.gson.annotations.SerializedName

class WordsUpdateResponse {
    @SerializedName("update")
    var isUpdate = false

    @SerializedName("revision")
    var revision = 0

    @SerializedName("data")
    var data: List<ThemeResponse>? = null
}